js-miniapp-sample/package.json

```diff
@@ -22,11 +22,12 @@
     "prettify": "prettier --config ./prettier.config.js --check src/**/*.js",
     "lint": "eslint src/ --fix-dry-run",
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build && npm run zip",
     "dockerBuild": "cd ci && make build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "flow": "flow"
+    "flow": "flow",
+    "zip": "(cd build; find . -name \"*.txt\" -type f -delete; zip -r js-miniapp-sample.zip ./*)"
```

Comment:
Nice one. Can we also add the command to remove hidden folder of mac?

Reply:
Good idea! I've update this.